mkdocs.yml

```diff
@@ -72,6 +72,7 @@ nav:
     - user-guide/upgrade.md
     - user-guide/documentation.md
     - user-guide/observability.md
+    - user-guide/scalability.md
```

Comment:
would it fit better under the 'Reference' section?

Reply:
Done in b39b1c767cb96acfb5ae2d986309368bbd1f026b